pkgs/by-name/co/cosmopolitan/package.nix

```diff
@@ -27,6 +27,12 @@ stdenv.mkDerivation (finalAttrs: {
     })
   ];
 
+  postPatch = ''
+    sed -i '/^DOWNLOAD := $(shell build\/download-cosmocc.sh /d' Makefile
+    mkdir .cosmocc
+    ln -s ${cosmocc} .cosmocc/3.9.2
```

Comment:
So this is using the downloaded binary cosmocc to bootstrap, right? Nothing against that just clarifying